app/wizards/schools/register_mentor_wizard/wizard.rb

```diff
@@ -23,6 +23,11 @@ class Wizard < ApplicationWizard
             review_mentor_details: ReviewMentorDetailsStep,
             review_mentor_eligibility: ReviewMentorEligibilityStep,
             trn_not_found: TRNNotFoundStep,
+            mentoring_at_new_school_only: MentoringAtNewSchoolOnlyStep,
+            started_on: StartedOnStep,
+            previous_training_period_details: PreviousTrainingPeriodDetailsStep,
+            programme_choices: ProgrammeChoicesStep,
+            lead_provider: LeadProviderStep,
```

Comment:
all steps are kept in alphabetical order for quickly search and find it. Please, can we keep that order?